Azure/sonic-buildimage
# [Mellanox] Inject SDK libs dependency to pmon on Mellanox platform

**- What I did**
Inject SDK libs dependency to pmon image on Mellanox platform
Mount mlnx-syncd exposed SDK stuff to pmon container on Mellanox platform
Remove unused '-e' options for Mellanox syncd container

**- How I did it**
Change Mellanox platform make files to inject pmon container image dependency on SDK libs
modify image ctrl templates for mount SDK volumes to pmon and remove unused "-e" option

**- How to verify it**
check syncd SDK and pmon container, they worked as usual
call SDK API from pmon and also work. 

**- Description for the changelog**
file changed:

```
files/build_templates/docker_image_ctl.j2
platform/mellanox/rules.mk
```



**- A picture of a cute animal (not mandatory but encouraged)**
